contrib/testgen/gen_key_io_test_vectors.py

```
@@ -124,7 +122,8 @@
     else:
         suffix = bytearray(template[2])
 
-    val = b58encode_chk(prefix + payload + suffix)
+    assert len(prefix) == 1
```

Ruff (0.12.2)

Use of `assert` detected

(S101)